src/app/components/TableCellAge/__tests__/index.test.tsx

```diff
@@ -23,12 +24,12 @@ describe('TableCellAge', () => {
   })
 
   afterEach(() => {
-    jest.useRealTimers()
-    jest.resetAllMocks()
+    vi.useRealTimers()
+    vi.resetAllMocks()
   })
 
   it('should display relative time by default', () => {
-    ;(useLocalSettings as jest.Mock).mockReturnValue({
```

Comment:
This should had been using `jest.mocked(useLocalSettings)` to get correct types. Does vite have something like it